### PR TITLE
Prep for publishing as a library and adding to the Bundle

### PR DESCRIPTION
**Pull request by dastels**

### CODE REVIEW

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
looks good but maybe needs some traviscare

**Comment by ladyada** [comment]
ok ping me when its passin' travis and ill do another quick 👁

File: examples/pyoa_simpletest.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
+#pylint disable=wrong-import-position
```
Comment (ladyada) [inline examples/pyoa_simpletest.py]: dont disable this, fix it instead!

File: examples/pyoa_simpletest.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+
 import sys
 sys.path.append('/Adafruit_CircuitPython_PYOA')
```
Comment (ladyada) [inline examples/pyoa_simpletest.py]: you can remove this, its for testing

**Review by ladyada — CHANGES_REQUESTED** [review]

**Comment by dastels** [comment]
Sphinx is choking on displayio.  I have no idea how to deal with that.

**Comment by ladyada** [comment]
https://github.com/adafruit/Adafruit_CircuitPython_PyPortal/blob/b1718ca6e0cf2e68d267d31ff082f7743b5aae29/docs/conf.py#L23

File: examples/pyoa_simpletest.py (hunk 2 above)
Comment (dastels) [reply examples/pyoa_simpletest.py]: Fixed, and Travis is green. Have another look.

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
good! we can tweak more as you play w/the library :) please submit to the bundle

**Pull request merged**